app/controllers/api/base_controller.rb

```diff
@@ -1,3 +1,22 @@
 class Api::BaseController < ApplicationController
   layout nil
-end
+
+  def authenticate_user
+    email = params[:email]
```

Comment:
Consider using environment variables for the JWT secret key instead of hardcoding it directly in the code. This enhances security by keeping sensitive information out of the source code.